frontend/src/pages/Teachers.vue

```diff
@@ -67,7 +67,12 @@
           </template>
         </b-table-column>
 
-        <b-table-column field="status" label="Status" searchable sortable>
+        <b-table-column
+          field="status.Description"
+          label="Status"
+          searchable
```

Comment:
```suggestion
          searchable
          :custom-search="(object, input) => object.status.Description.toLowerCase().startsWith(input.toLowerCase())"        
```

Reply:
Thank you for catching that! I've just pushed a commit incorporating your code :D